### DISCUSSION TITLE
How to group by date only in DateTime

### DISCUSSION THREAD

**Discussion by bangbaew** [prisma/prisma#11692] [Q&A] [answered]
I've tried using group by DateTime and it groups the time as well, so it returns individual records like this.

<img width="602" alt="image" src="https://user-images.githubusercontent.com/29559559/152977346-f80363f6-9f07-40cf-b1b8-8497eaeef8a1.png">

this is my code:
<img width="429" alt="image" src="https://user-images.githubusercontent.com/29559559/152977697-ff45ce9f-549d-44c8-a70f-f369834c55ce.png">

How can I group by the date part only in DateTime column?

**Comment by Luke265** [comment, marked as answer]
As far as I know, there is no way to separate DateTime using Prisma API. So you might have to use raw sql, or store date and time separately.

There are couple issues related to this: https://github.com/prisma/prisma/issues/5560 and https://github.com/prisma/prisma/issues/5761 , you could add 👍 reaction to those issues, so they would get more attention.

**Reply by EnkhAmar** [reply]
We need this feature :)) Hope we will be able to do it soon.

**Reply by VictorCalderon** [reply]
Is this feature still in planning? If so, how can we help develop it? I am new to open source, I would love to contribute if possible.

**Reply by kulterryan** [reply]
is that feature planned?

**Reply by muhaimincs** [reply]
It's been almost two years

**Reply by ListonFermi** [reply]
1077 days and counting...

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
